stdlib/src/socket.rs

```diff
@@ -726,19 +726,22 @@ mod _socket {
     }
 
     #[pyfunction]
-    fn htonl(x: u32) -> u32 {
+    const fn htonl(x: u32) -> u32 {
```

Comment:
#[pyfunction] is always wrapped to be a python function. const on these function will do nothing.

... nothing bad to do it though